com.unity.netcode.gameobjects/Components/Messages/NetworkTransformMessage.cs

```diff
@@ -0,0 +1,114 @@
+using Unity.Netcode.Components;
+using UnityEngine;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// NetworkTransform State Update Message
+    /// </summary>
+    internal struct NetworkTransformMessage : INetworkMessage, INetworkSerializeByMemcpy
```

Comment:
Nitpick: INetworkSerializebyMemcpy is unnecessary since you're never directly memcpy-serializing the message.

Reply:
Removed! 
:godmode: